CHANGELOG.md

```diff
@@ -45,9 +62,26 @@ New modules
   Algebra.Properties.IdempotentCommutativeMonoid
   ```
 
+* Refactoring of the `Algebra.Solver.*Monoid` implementations, via
+  a single `Tactic` module API based on the existing `Expr`, and
+  a common `Normal`-form API:
```

Comment:
I would add that you're still supposed to access the solvers through the existing files.